cmake.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/MediaInfoLib/Project/CMake/CMakeLists.txt b/MediaInfoLib/Project/CMake/CMakeLists.txt
+index 895965c2..e9d11ecd 100644
+--- a/MediaInfoLib/Project/CMake/CMakeLists.txt
++++ b/MediaInfoLib/Project/CMake/CMakeLists.txt
+@@ -85,11 +85,6 @@ elseif(APPLE)
+   set(CMAKE_DEBUG_POSTFIX "_debug")
+ endif()
+ 
+-# use bundled tinyxml only if no system

```

Comment:
You can probably do `-DCMAKE_DISABLE_FIND_PACKAGE_TinyXML` instead of patching CMake, right?